### PR TITLE
[10.0][MIG][partner_academic_title] partner academic title mgs

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**

### CODE REVIEW

File: partner_academic_title/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/partner-contact/issues/new?body=module:%20partner_academic_title%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (leemannd) [inline partner_academic_title/README.rst]: Can be removed

File: partner_academic_title/data/partner_academic_title_data.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
```
Comment (leemannd) [inline partner_academic_title/data/partner_academic_title_data.xml]: s/openerp/odoo/

File: partner_academic_title/views/res_partner_view.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+    <data>
```
Comment (leemannd) [inline partner_academic_title/views/res_partner_view.xml]: s/openerp/odoo/
data can be removed

File: partner_academic_title/views/partner_academic_title_view.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+    <data>
```
Comment (leemannd) [inline partner_academic_title/views/partner_academic_title_view.xml]: s/openerp/odoo/
data can be removed

**Review by leemannd — COMMENTED** [review]
Only a few details. Thanks for the port

**Comment by antoniocanovas** [comment]
This module works fine, but It creates a menu entry in "Contacts" which avoid the default full width list of contacts.
Default:
captura de pantalla 2017-05-29 a las 11 40 54

Tested on runbot:
captura de pantalla 2017-05-29 a las 11 45 45




I propose change this menu to "Sales => Configuration => Contacts"
captura de pantalla 2017-05-29 a las 11 42 19

**Comment by rafaelbn** [comment]
@GillesTephaneMeyomesse , please could you review and apply changes as commented in https://github.com/OCA/partner-contact/pull/393#issuecomment-304620532 thanks!

**Comment by GillesTephaneMeyomesse** [comment]
@rafaelbn , @antoniocanovas Thanks, changes will be applied on next commit

**Comment by rafaelbn** [comment]
Hi @GillesTephaneMeyomesse are you going to make changes? we are reviewing now in OCA Code sprint! 😄

**Comment by GillesTephaneMeyomesse** [comment]
Hi @rafaelbn Yes i will make changes, sorry for my late reply^^

**Comment by GillesTephaneMeyomesse** [comment]
Hi @rafaelbn , @antoniocanovas I'm not sure it's a good idea to create a dependency with the sales module ... I think this problem does not come from the partner_academic_title module. I have the same problems locally when passing to debug mode while the partner_academic_title module is not installed. I need to investigate a little more ...

**Comment by GillesTephaneMeyomesse** [comment]
Hi @antoniocanovas and @rafaelbn I think the bug is solved.
The group_system is defined in the partner_contact_configuration module and displays the "Contact" submenu for users in this group. So no need this group in the menuitem defined in partner_academic_title module.
I noticed that the group_system is evaluated when the developer mode is enabled. So when developer mode is activated and the group_system we will see the "Academic Title" menu and if we deactivate developer mode we will not see it again. (You may have to refresh the browser)
Can you test this ?

**Comment by pedrobaeza** [comment]
Please rebase to get a possible green branch

**Comment by GillesTephaneMeyomesse** [comment]
@pedrobaeza This branch has been rebased

**Comment by pedrobaeza** [comment]
It would be good also to have all the "OCA transbot" commits together (with the one removing the en.po file).

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍 thanks!

**Comment by GillesTephaneMeyomesse** [comment]
@pedrobaeza The OCA transbot commits have been squashed

**Pull request merged**